src/request.ts

```diff
@@ -78,6 +80,9 @@ export async function request({
         ...(isPost
           ? { "Content-Type": "application/json; charset=utf-8" }
           : undefined),
+        ...(user_agent
+          ? { "User-Agent": user_agent, "X-User-Agent": user_agent }
+          : undefined),
```

Comment:
@im-adithya should we do the fallback here instead so it's impossible not to pass a user agent?

Reply:
+1

Reply:
Done!